src/pages/Home.jsx

```diff
@@ -0,0 +1,43 @@
+import { Link } from "react-router-dom";
+import beers from "../assets/beers.png";
+import newBeer from "../assets/new-beer.png";
+import randomBeer from "../assets/random-beer.png";
+
+function Home() {
+  return (
+    <div>
+      <Link to={"/beers"}>
+        <img src={beers} alt="beer" />
+        <h2>All Beers</h2>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Esse ad
+          repudiandae tenetur fuga deleniti porro vero ducimus reiciendis
+          distinctio saepe, reprehenderit dolore delectus! Voluptate saepe eaque
+          quisquam eius harum assumenda?
+        </p>
+      </Link>
+
+      <Link to={"/randomBeer"}>
+        <img src={randomBeer} alt="random beer" />
+        <h2>Random Beer</h2>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellendus
+          corrupti veritatis ipsum autem esse atque a quae quasi distinctio,
+          modi non hic iste neque aliquam culpa amet est consectetur doloribus.
+        </p>
+      </Link>
+
+      <Link to={"/newBeer"}>
```

Comment:
This route does not exist. On app.js you defined it as /random-beer but when I replaced it started working again